src/bufr/BufrParser/BufrDescription.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include <iostream>
+
+#include "oops/util/IntSetParser.h"
+#include "eckit/exception/Exceptions.h"
+
+#include "BufrDescription.h"
+#include "BufrMnemonicSet.h"
+#include "BufrTypes.h"
+
+#include "Exports/MnemonicExport.h"
+#include "Exports/DatetimeExport.h"
+#include "Exports/Export.h"
+
+
+static const char* FILENAME = "obsdatain";
```

Comment:
Two options here:

1) Use an [unnamed / anonymous namespace](https://www.learncpp.com/cpp-tutorial/unnamed-and-inline-namespaces/) for these. No need to make them static, and you can ensure that the symbols are never accidentally accessed outside of this source file.

2) These are never used outside of ```BufrDescription::BufrDescription```. Why not define them there?

Reply:
Also, ALL_CAPS is usually reserved for macros. Cpplint might complain in the future.

Reply:
I was wondering if there was a naming convention that JCSDA follows for constant variables like this?

Reply:
Using anonymous namespaces with sub namespaces... Please take a look... in BufrDescription, IodaDescription, DatetimeExport..